### PR TITLE
Apple

### PR DESCRIPTION
**Pull request by yirano**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    </div>
+  </main>
+  <aside>
+    <div class="aside-img">
+      <img src="../imgs/img-1.jpg" alt="first image" />
+      <img src="../imgs/img-2.jpg" alt="second image" />
+      <img src="../imgs/img-3.jpg" alt="third image" />
+      <img src="../imgs/img-4.jpg" alt="fourth image">
+    </div>
+  </aside>
```
Comment (maelfosso) [inline src/index.html]: Could you move it into the `<main>` tag ?

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      <h3>Watch the "Your Verse" film and explore the stories </h3><img src="../imgs/play-button.svg" alt="play icon" style="width: 25px; -webkit-filter: invert(100%); filter: invert(100%); " class="play-icon">
+      </div>
+    </div>
+  </main>
+  <aside>
+    <div class="aside-img">
+      <img src="../imgs/img-1.jpg" alt="first image" />
+      <img src="../imgs/img-2.jpg" alt="second image" />
+      <img src="../imgs/img-3.jpg" alt="third image" />
+      <img src="../imgs/img-4.jpg" alt="fourth image">
```
Comment (maelfosso) [inline src/index.html]: All these images are links

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,154 @@
+* {
+  margin: 0;
+}
+
+body {
+  margin: 0;
```
Comment (maelfosso) [inline src/style.css]: Why that property ? You've already set `margin: 0;` up.

**Review by maelfosso — CHANGES_REQUESTED** [review]
Hello @yirano 

Thank for that web-page :smile: It's so beautiful.
Even if the result it's beautiful there are some things in the code that should change.

- [ ] The 4 bottom images are links. Please use `<a>` tags.

- [ ] The background image you used is not the same as the one of the Apple website archive. Why ?

- [ ] Please, could you move the `<aside>` into the `<main>` ?

- [ ] **README**: Please check the link of the live, it's not working.

Kindly make the changes and submit a new code review request.
Thank you.

File: src/index.html (hunk 1 above)
Comment (yirano) [reply src/index.html]: I'll change the <main> tag to <section> instead as to keep the same flow of the layout without additional lines of codes.

File: src/style.css (hunk 3 above)
Comment (yirano) [reply src/style.css]: Thank you for pointing this out!

**Review by Forison — COMMENTED** [review]
# Pull request under review by Addo

**Review by Forison — CHANGES_REQUESTED** [review]
Hi @Tiffany, 👋👋
 You have done a good job with this project🙂, but here are a few modifications I want you to do to have your project approved.😔
#### 1. **[REQUIRED]** navbar links🙂
The elements in the navbar have to be styled to look like buttons.
> Wrap the element in  ``` <a>``` tag. ie ```<nav><li><a>store</a></li><nav>``` and style it by setting text-decoration to none and add a hover effect the ``` <li>```🙂
#### 2. **[REQUIRED]** Main 🙂
As it stands now your page has a horizontal scroll which makes it not look good. The background image in overflowing my screen width. 🙂🙂
>You have set the width of the ```<main>``` to ```1440px``` as results the width would be fixed on different screen width so change it to 100%. and remove the ```padding``` set on it as well.
#### 2. **[REQUIRED]**  ```.aside-img```🙂
As it stands now your page has a horizontal scroll which makes it not look good. The background image in overflowing my screen width. 🙂🙂
>You have set the width of the ```.aside-img``` to ```1440px``` as results, the width would be fixed on different screen width so change it to 100%. and ```aside-img img``` to ```cal(100% / 4``` set on it as well.

> 🤔🤔🤔 ** [Am guessing when you inspect the page you get a width of 1440px. That is because that is the 100% of your screen size in pixel }**

#### 3. **[REQUIRED]** Bottom images 🙂
Each of the images is a link, so they ought to redirect users when they click them
> Wrap each of the four images in the ```aside-img```  in  ```<a href="#" >  </a>```
**[OPTIONAL]**
The use of  ```<aside><aside>``` in the context of this project makes it confusing you equally wrap it in a ```div``` tag with class name .```bottom``` 
do the various corrections I have suggested and you will be good to go.
#### Rectify and submit for review again. 🙂
# Happy coding 😇😇😇

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Quicksand:300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li><img src="../imgs/apple-icon.png" alt="apple icon" style="width: 20px; height: 17px;" /></li>
+        <li>Store</li>
```
Comment (Forison) [inline src/index.html]: # wrap the store,apple etc.. in a tag and style as suggested above.🙂

File: src/index.html [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+      <h2>Everyone has something to share. See how people are using iPad in their unique ways to add to the world's story.</h2>
+
+      <div class="h3-wrapper">
+      <h3>Watch the "Your Verse" film and explore the stories </h3><img src="../imgs/play-button.svg" alt="play icon" style="width: 25px; -webkit-filter: invert(100%); filter: invert(100%); " class="play-icon">
+      </div>
+    </div>
+  </main>
+  <aside>
+    <div class="aside-img">
+      <img src="../imgs/img-1.jpg" alt="first image" />
```
Comment (Forison) [inline src/index.html]: # wrap ```img```  tag in link tag as suggested in my main review 🙂

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+  height: 20px;
+  border: 1px solid rgb(102, 102, 102);
+  border-radius: 50px;
+  box-shadow: inset 0 1px 0 1px rgb(43, 43, 43);
+  background-color: rgba(255, 255, 255, 0.1);
+}
+
+main {
+  height: 900px;
+  width: 1240px;
```
Comment (Forison) [inline src/style.css]: # change width to 100% as suggested in the review

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+}
+
+.play-icon {
+  float: right;
+  padding-left: 8px;
+  margin-top: 8px;
+}
+
+aside {
+  width: 1440px;
```
Comment (Forison) [inline src/style.css]: # change width as suggested in my main review

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+  margin: 0 auto;
+}
+
+.aside-img img {
+  width: calc(1440px / 4);
```
Comment (Forison) [inline src/style.css]: # change value to 100% instead of 1440px;

**Review by Forison — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 5 above)
Comment (yirano) [reply src/index.html]: I was so focused on making this _look_ like the source that I had forgotten to turn it into a link. Thank you so much for pointing this out. I'll be sure to keep a more detailed look next time!

**Pull request closed without merging**